modules/kmm-symbolic-links-for-in-tree-dependencies.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="kmm-symbolic-links-for-in-tree-dependencies_{context}"]
+
+= Symbolic links for in-tree dependencies
+
+Some kernel modules depend on other kernel modules that are shipped with the node's operating system.
+To avoid copying those dependencies into the kmod image, KMM mounts `/usr/lib/modules` into both the build and the
+worker pod's filesystems.
+
+By creating a symlink from `/opt/usr/lib/modules/[kernel-version]/[symlink-name]` to
+`/usr/lib/modules/[kernel-version]`, `depmod` can use the in-tree kmods on the building node's filesystem to resolve
+dependencies.
+
+At runtime, the worker pod extracts the entire image, including the `[symlink-name]` symbolic link.
+That link points to `/usr/lib/modules/[kernel-version]` in the worker Pod, which is mounted from the node's filesystem.
```

Comment:
Slight suggestion. Also same sentiments as above. This is the last time that I'll point it out if you opt to change based on the RHSSG. 
```suggestion
That symbolic link points to `/usr/lib/modules/<kernel_version>` in the worker pod, which is mounted from the node's filesystem.
```

Reply:
Made all changes.